test/cql-pytest/util.py

```diff
@@ -179,6 +179,14 @@ def cql_session(host, port, is_ssl, username, password):
         # so let's increase them to 60 seconds. See issue #11289.
         connect_timeout = 60,
         control_connection_timeout = 60,
+        # The default timeout for new connections is 5 seconds, and for
+        # requests made by the control connection is 2 seconds. These should
+        # have been more than enough, but in some extreme cases with a very
+        # slow debug build running on a very busy machine, they may not be.
+        # so let's increase them to 60 seconds. See issue #11289.
+        connect_timeout = 60,
+        control_connection_timeout = 60,
+        max_schema_agreement_wait=60,
```

Comment:
As you can see just a few lines above, this file already had these parameters :-)
Except the "max_schema_agreement_wait" - I don't know what this one is about, and the commit message doesn't explain why it's beneficial, and since cql-pytest is about single-node clusters, I don't think schema agreement is even an issue.
So maybe no need to change this file as well?

Reply:
Fixed in https://github.com/scylladb/scylladb/commit/c1ebd7e95589bb313679f526c10045f6a1dcbe81